releases/latest.releases.yaml

```diff
@@ -195,3 +195,7 @@ repositories:
     type: git
     url: https://github.com/robotology/gym-ignition.git
     version: v1.2.1
+  matlab-whole-body-simulator:
+    type: git
+    url: https://github.com/dic-iit/matlab-whole-body-simulator.git
+    version: v2.0.0.rc1
```

Comment:
Did you checked if this will version will work fine with conda?

Reply:
How can I test the if it works with conda? By following the first two steps in https://github.com/robotology/robotology-superbuild/blob/master/doc/developers-faqs.md#how-to-do-a-new-release?

Reply:
You can trigger a job to build the conda packages without uploading them in the Action tab, see:

![image](https://user-images.githubusercontent.com/1857049/115594959-a4695680-a2d6-11eb-9089-00de63897157.png) 

I created one in https://github.com/robotology/robotology-superbuild/actions/runs/771480042 , but it will take some time before completing. Otherwise you can just use a release number without any string at the end, that has been work great for all other repos.

Reply:
Ah great, initially I thought that workflow_dispatch event trigger was specific to updating the releases.

Reply:
The conda packages generation was successful. The tag was not on the latest commit though, so I have to push the tag again. I guess there is no simple way to download and test those packages...

Reply:
> The conda packages generation was successful. The tag was not on the latest commit though, so I have to push the tag again. 

I think we wanted to only check if the ` v2.0.0.rc1` tag was processed correctly, so I think the test served its purpose. 



> I guess there is no simple way to download and test those packages...

You can generate locally using https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-recipe-generation.md, but I think the additional overhead is probably not worth it, I think we can release 2.0.0, upload the conda packages, and if we found problems fix them and upload 2.0.1, etc etc.